latte/dataset/synthetic.py

```diff
@@ -0,0 +1,107 @@
+from typing import Callable, List
+import dataclasses
+
+
+import numpy as np
+import geoopt
+
+
+rng = np.random.default_rng(42)
```

Comment:
In almost every case it's better to have an argument passed explicitly, rather than a global variable. (Especially that whenever we call it we modify it).


You can pass this `rng` as a default one, if it's not provided. (And the pattern for this is simple – you can pass it as an argument to `default_rng()` )

Reply:
Good point! I saw you've done it that way yes. Thanks :)

Reply:
If it is me who introduced this design anti-pattern, I'll need to issue formal apology letter :sweat_smile: